astrosat_users/adapters.py

```diff
@@ -306,8 +306,8 @@ def send_mail(self, template_prefix, email, context, **kwargs):
                 raise TypeError('"bcc" argument must be a list or tuple')
             self.bcc = list(bcc)
         msg = self.render_mail(template_prefix, email, context)
-        msg.cc = list(cc)
-        msg.bcc = list(bcc)
+        msg.cc = [address for address in bcc if address != email]
```

Comment:
Should this be for all addresses in `bcc`? I expected it to be for addresses in `cc`

Reply:
Oops.  Good catch.  I was working too fast.